### ISSUE TITLE
DocComment sniff reporting argument-less tags in DocBlock as error

### ISSUE THREAD

**Issue by aik099** [squizlabs/PHP_CodeSniffer#343]
Example code (class DocBlock, but this can be in any other DocBlock):

``` php
/**
 * Annotation for describing element selector in Selenium style.
 *
 * @link http://bit.ly/qa-tools-find-by-annotation
 * @usage('class'=>true, 'property'=>true, 'inherited'=>true, 'multiple'=>true)
 */
```

The line with `@link` is reported as indented incorrectly. I believe that argument-less tags, like `@internal` and any other unrecognized tag, like `@usage` in this case shouldn't force all other tags in same DocBlock to be indented around it.
